nsdb-cluster/src/main/resources/application-common.conf

```diff
@@ -61,3 +65,20 @@ akka {
 
   http.server.idle-timeout = 1 hour
 }
+
+akka.cluster.downing-provider-class = "com.swissborg.lithium.DowningProviderImpl"
+
+com.swissborg.lithium {
+  active-strategy = "keep-majority"
+
+  # Duration during which the cluster must be stable before taking
+  # action on the network-partition. The duration must chose large
+  # enough to allow for membership events to be gossiped and persistent
+  # actor to be migrated.
+  stable-after = 60s
+
+  keep-majority {
+    # Only take in account nodes with this role.
+    role = ""
```

Comment:
out of curiosity. Have you tried to remove completely the config, or is it required even if it's empty?